application/frontend/controllers/UserController.php

```diff
@@ -4,17 +4,49 @@
 use common\models\User;
 use frontend\components\BaseRestController;
 use Yii;
-use yii\web\NotFoundHttpException;
 
 class UserController extends BaseRestController
 {
+    public function actionIndex() // GET /user
+    {
+        return User::find()->all();
+    }
+
+    public function actionView($employeeId) // GET /user/abc123
+    {
+        $user = User::findOne(['employee_id' => $employeeId]);
+
+        if (isset($user)) {
+            return $user;
+        }
+
+        Yii::$app->response->statusCode = 204;
+    }
+
     public function actionCreate(): User
     {
-        $existingUser = User::findOne([
-            'employee_id' => Yii::$app->request->getBodyParam('employee_id')
-        ]);
+        $user = new User();
+
+        $user->scenario = User::SCENARIO_NEW_USER;
+
+        $user->attributes = Yii::$app->request->getBodyParams();
+
+        parent::save($user);
```

Comment:
no, I think that makes sense.  Polymorphism would suggest `$this->save` is correct, I believe.  I probably did that because I was working on the model so much...but that distinction about using `parent::` while you're in an overriding method is helpful.  https://github.com/silinternational/idp-id-broker/commit/b8a62cb0a0a485b152c2fc7b6cb775cb92a1599a